pyproject.toml

```diff
@@ -7,7 +7,7 @@ allow-direct-references = true
 
 [project]
 name = "multiversx-sdk-network-providers"
-version = "0.12.1"
+version = "0.12.2"
```

Comment:
Should be all-right, since it's a breaking, but fixing change.

Reply:
Technically breaking, actually fixing 😄